parallels_darwin.go

```diff
@@ -612,25 +642,44 @@ func (d *Driver) diskPath() string {
 	return absDiskPath
 }
 
+func (d *Driver) getShareFolders() (map[string]string, error) {
+	stdout, _, err := prlctlOutErr("list", "--info", d.MachineName)
+	if err != nil {
+		return nil, err
+	}
+
+	// Parse Shared Folder name (ID) and path
+	res := make(map[string]string)
+	for _, match := range reSharedFolder.FindAllStringSubmatch(string(stdout), -1) {
+		sName := match[1]
+		sPath := match[2]
+		log.Debugf("Found the configured shared folder. Name: %q, Path: %q\n", sName, sPath)
+		res[sName] = sPath
+	}
+	return res, nil
+}
+
+// Mounts shared folder to the specified guest path. It is assumed that host and guest paths are the same
 func (d *Driver) mountShareFolder(shareName string, mountPoint string) error {
 	// Check the host path is available
 	if _, err := os.Stat(mountPoint); err != nil {
 		if os.IsNotExist(err) {
-			log.Infof("Host path '%s' does not exist. Skipping mount to VM...", mountPoint)
+			log.Infof("Host path %q does not exist. Skipping mount to VM...", mountPoint)
 			return nil
 		}
 		return err
 	}
 
-	// Ensure that share is available on the guest side
-	checkCmd := "sudo modprobe prl_fs && grep -w " + shareName + " /proc/fs/prl_fs/sf_list"
+	// Ensure that the share is available on the guest side
+	checkCmd := fmt.Sprintf("sudo modprobe prl_fs && grep -w %q /proc/fs/prl_fs/sf_list", shareName)
 	if _, err := drivers.RunSSHCommandFromDriver(d, checkCmd); err != nil {
-		log.Infof("Shared folder '%s' is unavailable. Skipping mount to VM...", shareName)
+		log.Infof("Shared folder %q is unavailable. Skipping mount to VM...", shareName)
 		return nil
 	}
 
-	// Mount shared folder
-	mountCmd := "sudo mkdir -p " + mountPoint + " && sudo mount -t prl_fs " + shareName + " " + mountPoint
+	// Mount the shared folder
+	log.Infof("Mounting shared folder %q ...", mountPoint, mountPoint)
+	mountCmd := fmt.Sprintf("sudo mkdir -p %q && sudo mount -t prl_fs %q %q", mountPoint, shareName, mountPoint)
```

Comment:
Why don't you use "share" option of prl_fs?

Reply:
Hm... I'm not sure. It was always like that, without `share` mount option. Do you think we should use it? Won't it cause access issues for docker daemon or containers running in the VM?

Reply:
Well, I don't know. Ok, anyway this is not relevant for the pull request.